Client/Frontend/Strings.swift

```diff
@@ -3621,6 +3621,21 @@ extension String {
             tableName: "Shopping",
             value: "Try Again",
             comment: "Button text of the error displayed in the shopping review quality bottom sheet.")
+        public static let HighlightsCardTitle = MZLocalizedString(
+            key: "", // Shopping.HighlightsCard.Title.v119
```

Comment:
I'm sure I'm missing context, but what's happening here?

Reply:
The strings should not be exported yet because they are not final. Using an empty key omits them from the export.